gdal/frmts/jpegxr/frmt_jpegxr.html

```diff
@@ -0,0 +1,80 @@
+<html>
+<head>
+<title>JPEGXR -- JPEG XR File Format</title>
+</head>
+
+<body bgcolor="#ffffff">
+
+<h1>JPEGXR -- JPEG XR File Format</h1>
+
+<p>The JPEG XR format is supported for reading, and batch writing, but not
+update in place.  JPEG XR files are represented as one band (Greyscale) or three
+band (RGB) or four band (RGB with Alpha) datasets with Byte valued bands.<p>
+
+<p>The driver is implemented based on the 
+    <a href="https://jxrlib.codeplex.com/">jxrlib</a> - open source (BSD)
+    implementation of JPEG XR format by Microsoft.</p>
+
+<h2>Driver Features</h2>
+
+<ul>
+<li>Compression - lossless by default, controlled with <code>QUALITY</code> option.</li>
+<li>Image Structure - entire image is decoded; tiling not yet supported.</li>
+<li>Channel Organization - Grey, RGB, BGR, RGBA, BGRA.</li>
+<li>Color - RBG images YUV with 8-bit per sample and
+<a href="https://en.wikipedia.org/wiki/Chroma_subsampling">4:4:4</a> full chroma
+resolution encoding by default.</li>
+</ul>
+
+
```

Comment:
@schwehr No reason. Perhaps I copied the file from `jpegls/frmt_jpegls.html` (w/ 32-33 lines blank) or other .html in GDAL, don't remember. I will clean it up.